duckduckgo/iOS
# Add default browser pushes

Task/Issue URL: https://app.asana.com/0/1187352151074490/1192933920238457/f
Tech Design URL:
CC: @brindy 

**Description**:
Contains three separate pushes for users to change their default browser (all three are iOS 14 only):
- Swaps out the home row onboarding flow on iOS 14 to a new default browser onboarding flow
- Adds a row to settings in the app to change your browser (opens the settings app)
- Adds a message on the home screen (that should only show for users that haven't seen the new onboarding flow)

**Steps to test this PR**:
1. Install the app from scratch and test the new onboarding flow works on iOS 14, and that it doesn't appear on older versions of iOS
2. Open settings and test the default browser button. Do like above for different iOS versions
3. Test home message appears for existing users on iOS 14.
4. Test home message doesn't appear for users that have seen the new onboarding flow
5. Test home message isn't displayed once it's been dismissed/the button has been interacted with
6. Test the home message when the user both does and doesn't have favourites
7. Test the home message sizing works on the smallest/largest devices 
8. Test the home row reminder is now shown based on when the home message was dismissed



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
